tools/lint/lint.py

```diff
@@ -624,6 +624,23 @@ def check_script_metadata(repo_root, path, f):
     return errors
 
 
+def check_multiple_identical_values(repo_root, path):
```

Comment:
So, this function will be passed the `repo_root` and a list of all the paths, not a single path.